packages/apollo-server-plugin-response-cache/src/index.ts

```diff
@@ -1,3 +1,4 @@
 import plugin from './ApolloServerPluginResponseCache';
 export default plugin;
+export { plugin as ApolloServerPluginResponseCache };
 module.exports = plugin;
```

Comment:
Actually, there is something weird here. Why does this file have both `export default` and `module.exports =`? That's the problem, which I'll fix.